docs/tutorials/09_saving_and_loading_models.ipynb

```diff
@@ -563,7 +554,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": 19,
```

Comment:
In "Load a Model and Continue training" this needs updating

>  We also set quantum instance of the underlying network to the statevector simulator we created in the beginning of the tutorial.

Reply:
Replaced with "We also set the sampler property of the underlying network to the second instance of the Sampler primitive we created in the beginning of the tutorial."